src/test/scala/com/cognite/sdk/scala/v1/DataSetsTest.scala

```diff
@@ -0,0 +1,96 @@
+package com.cognite.sdk.scala.v1
+import java.time.Instant
+
+import com.cognite.sdk.scala.common._
+import com.cognite.sdk.scala.common.SetValue
+
+class DataSetsTest extends SdkTestSpec {
```

Comment:
Can't we use any of the standard tests? "behaves like readable" etc.?

Reply:
Nope. I get the 405 error.

Reply:
But like I said, we can implement list in terms of filter, you just need to call that instead of attempting to use the default implementation of list.